backend.py

```diff
@@ -907,15 +906,16 @@ def update_payment_renewal_info(sql_conn:                 sqlite3.Connection,
 
     if not result:
         payment_id = payment_tx.google_order_id if payment_tx.provider == base.PaymentProvider.GooglePlayStore else payment_tx.apple_tx_id
-        err.msg_list.append(f'Updating payment TX failed, no matching payment found for {payment_tx.provider.name} {payment_id}')
+        err.msg_list.append(f'Updating payment TX failed, no matching payment found for {payment_tx.provider} {payment_id}')
     return result
 
-def add_unredeemed_payment(sql_conn:              sqlite3.Connection,
-                           payment_tx:            PaymentProviderTransaction,
-                           plan:                  ProPlanType,
-                           expiry_unix_ts_ms:     int,
+def add_unredeemed_payment(sql_conn:            sqlite3.Connection,
+                           payment_tx:          PaymentProviderTransaction,
+                           plan:                ProPlanType,
+                           expiry_unix_ts_ms:   int,
                            unredeemed_unix_ts_ms: int,
-                           err:                   base.ErrorSink):
+                           platform_refund_expiry_ts_ms: int,
```

Comment:
This new field breaks the apple tests, the apple tests are runnable now without any changes/configuration so it should show up if you run the full test suite

Reply:
fixed and passing now